utilities/blob_db/blob_file.h

```diff
@@ -98,19 +104,25 @@ class BlobFile {
 
   bool header_valid_;
 
+  bool footer_valid_;
```

Comment:
Wondering if these is a need for these to be made atomic?

Reply:
I'll cleanup mutex usage and the BlobFile interface in the next PR.